src/components/input-group/input-group.stories.mdx

```diff
@@ -0,0 +1,16 @@
+import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
+import inputGroupDemo from './demo/input-group-demo.twig';
+import './input-group.scss';
+import './demo/styles.scss';
+
+<Meta title="Components/Input Group" />
+
+## Input Group
+
+The Input Group combines the [Button](/story/components-button--button-element) and [Input](/story/components-input--text-elements) components. This would work well in creating design elements like search bars.
+
+Buttons and inputs can be put in whatever order you choose, and the components will adjust accordingly.
+
+<Canvas>
+  <Story name="Example">{(args) => inputGroupDemo(args)}</Story>
```

Comment:
Because this pattern uses blocks, we're going to need to do some more work to get a good code sample. See what @gerardo-rodriguez suggested to @dromo77 here: https://github.com/cloudfour/cloudfour.com-patterns/pull/1395#discussion_r669836582